ui/src/submissions/jobs/components/JobForm.jsx

```diff
@@ -111,6 +122,20 @@ class JobForm extends Component {
           placeholder="URL for additional information"
           component={TextField}
         />
+        {this.isPostDocSubmissions() && (
+          <FieldInfoAlert
+            description={
+              <span>
+                Many institutions have agreed to set January 7 as the earliest
+                deadline which can be imposed for accepting offers of
+                postdoctoral positions.{' '}
+                <ExternalLink href="//insti.physics.sunysb.edu/itp/postdoc-agreement.html">
+                  Learn More
+                </ExternalLink>
+              </span>
+            }
+          />
+        )}
```

Comment:
no update in the e2e or screenshot?

Reply:
yeah no, I didn't think it was worth do a whole SS for this.